build.sbt

```diff
@@ -641,28 +848,37 @@ val testSettings = {
   )
 }
 
-val jsTestSettings = {
-  Seq(
-    coverageEnabled := false, // Disable until Scala.js 1.0 support is there https://github.com/scoverage/scalac-scoverage-plugin/pull/287
-    doctestGenTests := {
-      Seq.empty
-    }
-  )
-}
+val jsTestSettings = Seq(
+  coverageEnabled := false, // Disable until Scala.js 1.0 support is there https://github.com/scoverage/scalac-scoverage-plugin/pull/287
+  doctestGenTests := Seq.empty
+)
+
+lazy val disabledSettings = Seq(
```

Comment:
Can you leave a comment on what `disabledSettings` does and how it works? It looks like it largely is for disabling for Scala 3?